### ISSUE TITLE
error with plot_cladogram.py

### ISSUE THREAD

**Issue by glucksfall** [SegataLab/lefse#2]
Hello,

I have found a problem running the plot_cladogram.py with the hmp_aerobiosis_small.txt test data:

```
Traceback (most recent call last):
  File "/opt/repositories/git-reps/lefse.SegataLab/plot_cladogram.py", line 340, in <module>
    clad_tree = read_data(params['input_file'],params)    
  File "/opt/repositories/git-reps/lefse.SegataLab/plot_cladogram.py", line 106, in read_data
    abundances = [float(v) for v in zip(*rows)[1] if v >= 0.0]
TypeError: 'zip' object is not subscriptable
```

Best regards.

PS: Also, the link http://huttenhower.sph.harvard.edu/webfm_send/129 shows a 404 error. I downloaded the hmp_aerobiosis_small.txt from https://raw.githubusercontent.com/xiucz/metagenome1/master/LEfSe/lefse/example/hmp_aerobiosis_small.txt

**Comment by glucksfall** [comment]
BTW, the full command was `plot_cladogram.py hmp_aerobiosis_small.res hmp_aerobiosis_small.cladogram.png --format png

**Comment by fbeghini** [comment]
Hi Rodrigo, yes, thank you for spotting this, we have forgot to unlist the zip content when porting to Python 3.
This have been fixed in commit 4eb6faa4518c55cb88b9933d08eea4dca0e9623f

Where is located the link you are referring to? You can download the same file from the LEfSE tutorial on bioBakery ([hmp\_small\_aerobiosis.txt](https://github.com/biobakery/biobakery/raw/master/demos/biobakery_demos/data/lefse/input/hmp_small_aerobiosis.txt)) 

Best

**Comment by glucksfall** [comment]
Hi @fbeghini,

Thanks for your reply. I found another error:

```
Traceback (most recent call last):
  File "../plot_cladogram.py", line 340, in <module>
    clad_tree = read_data(params['input_file'],params)    
  File "../plot_cladogram.py", line 106, in read_data
    abundances = [float(v) for v in list(zip(*rows))[1] if v >= 0.0]
  File "../plot_cladogram.py", line 106, in <listcomp>
    abundances = [float(v) for v in list(zip(*rows))[1] if v >= 0.0]
TypeError: '>=' not supported between instances of 'str' and 'float'
```

I fixed it with `abundances = [float(v) for v in list(zip(*rows))[1] if float(v) >= 0.0]`.

I forgot to say that I followed the bioconda-lefse_run.sh script in the example folder to run the tests. The link there also needs a fix.

I appreciate your time, 
Best regards.
